<?xml version="1.0"?>
<launch>
<!-- https://github.com/ADVRHumanoids/centauro_cartesio/tree/2.0-devel/launch -->
    
    <arg name="config_file"/>

    <arg name="gdb" default="false"/> <!-- for debug with gdb -->
    <let if="$(var gdb)" name="prefix" value="gdb -ex run --arg" />
    <let unless="$(var gdb)" name="prefix" value="" />

    <node pkg="estimation_utils" 
          exec="force_estimation_executor" 
          name="force_estimator"
          output="screen"
          launch-prefix="$(var prefix)">

        <param from="$(var config_file)"/>

        <remap from="force_estimator/robot_description" to="xbotcore/robot_description"/>
        <remap from="force_estimator/robot_description_semantic" to="xbotcore/robot_description_semantic"/>
        <remap from="~torque_offset" to="cartesian/torque_offset"/>
        <remap from="/xbotcore/command" to="null"/>
                    
    </node>

</launch>
